templates/repo/code/upstream_diverging_info.tmpl

```diff
@@ -0,0 +1,12 @@
+{{if and .UpstreamDivergingInfo .UpstreamDivergingInfo.CommitsBehind}}
+<div class="ui message flex-text-block">
+	<div class="tw-flex-1">
+		{{$upstreamLink := printf "%s/src/branch/%s" .Repository.BaseRepo.Link .BranchName}}
+		{{$upstreamHtml := HTMLFormat `<a href="%s">%s:%s</a>` $upstreamLink .Repository.BaseRepo.FullName .BranchName}}
+		{{ctx.Locale.TrN .UpstreamDivergingInfo.CommitsBehind "repo.pulls.upstream_diverging_prompt_behind_1" "repo.pulls.upstream_diverging_prompt_behind_n" .UpstreamDivergingInfo.CommitsBehind $upstreamHtml}}
+	</div>
+	<button class="ui compact green button tw-m-0 link-action" data-url="{{.Repository.Link}}/branches/merge-upstream?branch={{.BranchName}}">
```

Comment:
Need to escape branch name or is go template auto escape smart enough?

Reply:
It escapes correctly when the branch name is `a%b`

![image](https://github.com/user-attachments/assets/9d84f4db-de41-4a8b-88ee-fa6e1c528c4e)

Reply:
Hmm, the "printf" one needs to be escaped .... done in e4085c1ef1d5fdd7b7bcc0c8cc0cb35295837851